docs/publications/BioHackrXiv/paper.md

```diff
@@ -0,0 +1,161 @@
+While granularity supports the composability of tools into diverse purpose-specific workflows, this decentralised development and modular architecture make it difficult for Galaxy users to find and use tools. It may also result in Galaxy tool-wrapper developers duplicating efforts by simultaneously wrapping the same software. 
+This is further complicated by the lack of tool metadata, which prevents filtering of tools as relevant for a specific scientific community or domain, and makes it impossible to employ advanced filtering by ontology terms like ones from EDAM[@citesAsAuthority:black2021edam]. 
+The final challenge is also an opportunity: the global nature of Galaxy means that it is a big community. Solving the visibility of tools across this "ecosystem", and the potential benefits, are far-reaching for global collaboration on development of data-analysis tools and workflows.
+
+
+To provide the scientific community with a comprehensive list of available Galaxy tools, we developed a pipeline at the [ELIXIR BioHackathon Europe 2023](https://2023.biohackathon-europe.org) that collects Galaxy wrappers from a list of GitHub repositories and automatically extracts their metadata (including Conda version, bio.tools identifiers[@usesDataFrom:ison2016tools, @usesDataFrom:Ison2019], and EDAM annotations). The pipeline also queries the availability of the tools and usage statistics from the three main Galaxy servers (https://usegalaxy.eu, https://usegalaxy.org, https://usegalaxy.org.au).
+
+
+Crucially, the pipeline can filter its inputs to only include tools that are relevant to a specific scientific community. Based on the selected filters, a community-specific interactive table is generated that can be embedded, _e.g._ into the respective Galaxy Hub page or Galaxy subdomain. This table allows further filtering and searching for fine-grained tool selection. The pipeline is fully automated and executes on a weekly basis. Any scientific community can apply the pipeline to create a table specific to their needs.
+
+
+An interactive table that presents metadata is only as useful as the metadata annotations it is capturing. 
+To improve the metadata coverage for the interactive table, the project also directly addressed the quality of tool annotations in bio.tools for the microGalaxy community: a community with a focus on tools related to microbial research. 
+Annotation guidelines were established for this purpose, the process of updating Galaxy tool wrappers to include bio.tools identifiers was started and the outcome of these activities was evaluated using a crowdsourced approach. During the BioHackathon Europe week, the annotation practices were applied to the tools selected from the microGalaxy community. 
+This effort allowed the team to connect more than 50 tools to their respective bio.tools entry, update the registry entry, and peer-review the result. 
```

Comment:
Is this the correct number then? (I'm still not sure I got it right😅) Thanks @supernord 🙏🏽
```suggestion
This effort allowed the team to connect additional 46 tools to their respective bio.tools entry, update the registry entry, and peer-review the result. 
```

Reply:
@paulzierep